virttest/ovirt.py

```diff
@@ -9,10 +9,10 @@
 import logging
 
 try:
-    from ovirtsdk.api import API
-    from ovirtsdk.xml import params as param
+    import ovirtsdk4 as sdk
```

Comment:
suggest import different modules as per specific ovirt version, otherwise it will lose compatibility

Reply:
We will use ovirt4 in both python2 and python3 env, therefore no conflict

Reply:
rhv3.6 is going to out of service, it's not necessary to be compatible with an out of service product.